config/core/test-configs.yaml

```diff
@@ -1763,6 +1763,7 @@ test_configs:
       - baseline-nfs
       - kselftest-filesystems
       - kselftest-futex
+      - kselftest-rtc
```

Comment:
OK: https://staging.kernelci.org/test/plan/id/61e84bce6e7616b16d217804/
Interestingly, there are failures with the Clang build:
https://staging.kernelci.org/test/plan/id/61e848fa5d62af4ad321782f/

Maybe they are just intermittent issues, more tests would probably be needed to determine whether it's related to the compiler difference.

Reply:
Hmm the Clang log looks like it never even reached the login prompt, let alone run the test. Seems like an infrastructure issue (on either KernelCI or LAVA).